doc/rst/tools/mason/mason.rst

```diff
@@ -766,59 +780,50 @@ Once your package is uploaded, maintain the integrity of your package, and pleas
 chapel team if your package should be taken down.
 
 
+
+If you have a personal remote registry, ``mason publish <path-to-registry>``  also accepts
+a remote path to a git repository. This will create a branch to your registry that adds
+your package, and you can approve the PR to merge your new package into your registry.
+Must ensure your package has a remote origin in order to publish remotely.
+
+Publishing to a personal remote registry
+
+.. code-block:: sh
+   
+   cd PackageA
+   mason publish <remote-path-to-registry>
+
+To assess the ability of your package to be published to the mason-registry or
+a personal registry, run ``mason publish --dry-run <path-to-registry>`` for a
+series of quick checks or ``mason publish --check <path-to-registry`` for a more
+in depth check of the packages ability to be published.
+
 Local Registries
 ================
 
 It is sometimes desirable to use a local registry, for example with libraries
 you don't intend to distribute. The following steps create a local registry
-starting with Bricks for ``PackageA`` and ``PackageB`` which were created with
-``mason new PackageA`` and ``mason new PackageB``, and are located at
-``/path/to/my/packages/Package[AB]``. It is expected that mason will be
-extended to simplify and handle more of this process.
+starting with Bricks for ``PackageA`` which was created with ``mason new PackageA``
+Once you have successfully created a local registry, ``mason publish <path-to-local registry``
+can be used to publish automatically.
 
 First create, commit, and tag the packages that will be in the registry:
 
-.. code-block:: sh
-
-   # Create PackageA
-   cd /path/to/my/packages
-   mason new PackageA
-   cd PackageA
-   git add Mason.toml src/PackageA.chpl
-   git commit
-   git tag -a v0.1.0 -m "Tag version 0.1.0"
 
-   # Create PackageB
-   cd ..
-   mason new PackageB
-   cd PackageB
-   git add Mason.toml src/PackageB.chpl
-   git commit
-   git tag -a v0.1.0 -m "Tag version 0.1.0"
-
-Next, create a local registry:
+Create a local registry:
 
 .. code-block:: sh
 
    # Create the local registry
    mkdir /path/to/local/registry
    cd /path/to/local/registry
-   mkdir -p Bricks/PackageA Bricks/PackageB
-
-   # Add bricks for PackageA and PackageB
-   cp /path/to/my/packages/PackageA/Mason.toml Bricks/PackageA/0.1.0.toml
-   cp /path/to/my/packages/PackageB/Mason.toml Bricks/PackageB/0.1.0.toml
-
-   # Edit Bricks/PackageA/0.1.0.toml to add:
-   source = "/path/to/my/packages/PackageA"
-
-   # Edit Bricks/PackageB/0.1.0.toml to add:
-   source = "/path/to/my/packages/PackageB"
+   mkdir /Bricks/
```

Comment:
I suspect this example won't actually work because the `Bricks` directory can't be committed without an actual file in it, meaning the checkout of this registry will give an error about `Bricks` not being found.